src/mn_setup.c

```diff
@@ -1550,6 +1551,7 @@ static setup_menu_t weap_settings2[] = {
     {"9Th Choice Weapon", S_WEAP | S_BOOM, OFF_CNTR_X, M_SPC, {"weapon_choice_9"}},
     MI_GAP,
     {"Use Weapon Toggles", S_ONOFF | S_BOOM, OFF_CNTR_X, M_SPC, {"doom_weapon_toggles"}},
+    {"Full Weapon Cycle", S_ONOFF | S_BOOM, OFF_CNTR_X, M_SPC, {"full_weapon_cycle"}},
```

Comment:
I think these both need a better description now. How about "Same key toggles weapons" and "Prev/Next cycles through all weapons"? 🤷

Reply:
This is the best I could come up with, while still fitting in a 4:3 window: https://github.com/fabiangreffrath/woof/pull/1920/commits/e09f8dc4af0779de639e5cfc20aa3cb43063ca9c

Reply:
Oh, I was thinking about better description of the *menu items*.

Reply:
I was too, but "Use Weapon Toggles" and "Vanilla Weapon Cycle" are the best I can come up with while still fitting a 4:3 window.

Reply:
We could move the whole menu content a bunch to the right.

Reply:
Fixed here: https://github.com/fabiangreffrath/woof/pull/1920/commits/2d0692a421a0a14803401995cb63fdd5aecfe7b7

Reply:
A lot better, thank you!